jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramPresenter.java

```diff
@@ -46,6 +46,7 @@ public void setProcessImageService(final Caller<ProcessImageService> processImag
     public void onProcessSelectionEvent(@Observes final ProcessDefSelectionEvent event) {
         view.showBusyIndicator(constants.Loading());
         view.disableExpandAnchor();
+        view.setProcessDiagramDivId("ProcessDefDiagram");
```

Comment:
the presenter should not be responsible for defining an id in the view. Simply generated a ramdom one on the view side?
`DOM.createUniqueId();`

Reply:
if use the way, one page two "<div class="col-md-12" data-field="diagramContainerDiv" data-i18n-prefix="ProcessDiagramWidgetViewImpl.">", is it reasonable?

Reply:
updated, Please review again.

Reply:
sorry but I didnt get whats the issue here.